Sources/LiveViewNative/Modifiers/Navigation Modifiers/NavigationSubtitleModifier.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  NavigationSubtitleModifier.swift
+// LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/18/23.
+//
+
+import SwiftUI
+
+/// Configures the view’s subtitle for purposes of navigation. Only supported on macOS.
+///
+/// ```html
+/// <VStack modifiers={navigation_subtitle(@native, subtitle: "Subtitle Label")}>
+///     <Text>Top</Text>
+///     <Text>Bottom</Text>
+/// </VStack>
+/// ```
+///
+/// ## Arguments
+/// * ``subtitle``
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+struct NavigationSubtitleModifier: ViewModifier, Decodable {
```

Comment:
Can you add availability, this will be shown in the docs too:
```suggestion
@available(macOS 13.0, *)
struct NavigationSubtitleModifier: ViewModifier, Decodable {
```